common/JMeter-script-template/README.md

```diff
@@ -0,0 +1,31 @@
+## Introduction
+This repository contains a template for creating a JMeter test script. Below are placeholders to fill information into this README file. There is a sample JMX script that is set up with
+the standard required parameters and values, including those for credentials and a login API call. It also includes the jmeter-supported-data directory to place any data needed at runtime.
+There is also a scripts folder to hold any script to be run at runtime or pre/post testing. In addition to these basic folders, teams are welcome to create other folders to support their test
+as necessary.
+
+## Workflow steps
+##### Workflow for xyz.jmx
+Describe the workflow that this test models and necessary information about using this JMX file
+1. 
+2. 
+3. 
+
+
+## Modules required to be enabled as a pre-requisite to run JMeter script
+##### Backend:
+Back-end modules and version numbers. e.g.:
+- mod-circulation-19.0.9
+- okapi-3.1.1
+- ...
+##### Frontend:
+Front-end modules and version numbers. e.g.:
+- folio_circulation-3.0.1
+- ...
+
```

Comment:
Normally we mention the release. Like for what release this script intended to run for. for example:
`- FOLIO release: Goldenrod`

Reply:
Please check the latest commit. I put the line FOLIO release: Goldenrod on top of Backend and Frontend lists.